br/backup-and-restore-overview.md

```diff
@@ -0,0 +1,123 @@
+- **Encrypt backup data**: BR supports backup data encryption and amazon S3 server-side encryption. You can select an encryption method as required. For details, see [Encrypt backup data](/br/br-usage-backup.md#encrypt-backup-data).
+
+#### Impact on performance
+
+The impact of backup on a TiDB cluster is kept below 20%, and this value can be reduced to 10% or less with reasonable configuration of the TiDB cluster. The backup speed of a TiKV node is scalable and ranges from 50 MB/s to 100 MB/s. For more information, see [Backup performance and impact](/br/br-usage-backup.md#backup-performance-and-impact).
+
+#### Storage of backup data
+
+BR supports backing up data to Amazon S3, Google Cloud Storage, Azure Blob Storage, NFS, and other S3-compatible file storage services. For details, see [Back up data to external storage](/br/br-usage-backup.md#back-up-data-to-external-storage).
+
+### Restore TiDB cluster data
+
+This section describes restoration features and impact on performance.
+
+#### Restoration features
+
+- **Restore cluster snapshot backup**: You can restore snapshot backup data to a new cluster. For details, see [Restore TiDB cluster snapshots](/br/br-usage-restore.md#restore-tidb-cluster-snapshots).
+- **Restore cluster incremental backup**: You can restore the incremental backup data to a cluster. For details, see [Restore incremental backup](/br/br-usage-restore.md#restore-incremental-data).
+- **Restore a database or a table from backup**: You can restore part of a specific database or table. During the process, BR will filter out unnecessary data. For details, see [Restore a database or a table](/br/br-usage-restore.md#restore-a-database-or-a-table).
+
+#### Impact on performance
+
+Restoration proceeds with a scalable speed. Generally, the speed is 100 MB/s per TiKV node. BR only supports restoring data to a new cluster and uses resources of the target cluster as much as possible. For more details, see [Restoration performance and impact](/br/br-usage-restore.md#restoration-performance-and-impact).
+
+## Before you use BR
+
+Before you use BR, pay attention to its usage restrictions, compatibility, and other considerations.
+
+### Usage restrictions
+
+This section describes usage restrictions of BR.
+
+#### Unsupported scenario
+
+When BR restores data to the upstream cluster of TiCDC or TiDB Binlog, TiCDC or TiDB Binlog cannot replicate the restored data to the downstream cluster.
+
+#### Compatibility
+
+The compatibility issues of BR and a TiDB cluster are as follows:
+
+- There are some cross-version compatibility issues:
+
+    BR(< v5.4.0) cannot restore `charset=GBK` tables. In the meantime, no version of BR supports restoring `charset=GBK` tables to a TiDB cluster earlier than v5.4.0.
```

Comment:
Please avoid using `<` because it might be parsed as a tag.
```suggestion
    BR (< v5.4.0) cannot restore `charset=GBK` tables. In the meantime, no version of BR supports restoring `charset=GBK` tables to a TiDB cluster earlier than v5.4.0.
```